tenpy/algorithms/dmrg.py

```diff
@@ -170,12 +185,19 @@ class     A string stands for one of the engines defined in this module,
     -------
     info : dict
         A dictionary with keys ``'E', 'shelve', 'bond_statistics', 'sweep_statistics'``
+
+    Raises
+    ------
+    ValueError
+        If `n` is not set to `1` or `2`.
     """
     # initialize the engine
-    Engine_class = get_parameter(DMRG_params, 'engine', 'EngineCombine', 'DMRG')
-    if isinstance(Engine_class, str):
-        Engine_class = globals()[Engine_class]
-    engine = Engine_class(psi, model, DMRG_params)
+    if n ==  1:
```

Comment:
I think it might be convenient to have the `n` as a parameter inside the DMRG_params dictionary, especially to unify the setup with other algorithms (general call structure `Engine(psi, model, engine_pars)`), already thinking towards #46.
The way it was before, we had a single extra DMRG parameter 'engine', which could be a string naming the class inside this module, or the class itself, if the user derived the DMRG class to adjust some function for whatever reason.

Reply:
Done; used a parameter 'active_sites'.

This could lead to confusion when calling an engine manually, but we can expect anyone who does this to be experienced enough to not expect the TwoSiteDMRGEngine to be a 1-site algorithm if 'active_sites' = 1.